### ISSUE TITLE
[UMBREL] Will not launch UI

### ISSUE THREAD

**Issue by mokeybear** [bitcoinknots/bitcoin#95]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behaviour

Hello,

Fresh Umbrel install on RPI 5 latest Umbrel OS image 1.3 I attempted to launch by simply clicking on the icon and it attempts 2105 port and it errors out immediately. I checked logs and saw 3005 was listening so I attempted to connect to that and no dice. errors in Log

### Expected behaviour

App opens

### Steps to reproduce

1) Install Umbrel 1.3 latest for RPI 5 
2) Complete Setup
3) Install Bitcoin Knots from Umbrel application store
4) attempt to launch. 

### Relevant log output

Warning: APP_BITCOIN_KNOTS_INTERNAL_tor_PORT is unset or empty
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_app_proxy_1  | exec /usr/local/bin/docker-entrypoint.sh: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
bitcoin-knots_tor_1        | exec /usr/local/bin/tor: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error
tor_server-1               | exec /umbrel/entrypoint.sh: exec format error

### How did you obtain Bitcoin Knots

Package manager

### What version of Bitcoin Knots are you using?

Bitcoin Knots version v27.1.knots20240801

### Operating system and version

Umbrel 1.3.0

### Machine specifications

RPI 5 
8GB Ram

**Comment by luke-jr** [comment]
This looks like an Umbrel issue. @Retropex

**Comment by Retropex** [comment]
Did you install the package on the official AppStore? @mokeybear

**Comment by mokeybear** [comment]
> Did you install the package on the official AppStore? @mokeybear

Yes I did sorry for the delay. Its on an RPI5 and whenever I click the Application icon it just forwards to a 404 browser page.

**Comment by Retropex** [comment]
@nmfretz may I have some help on this please, I don't have a rpi5 but it should be aarch64 like rpi4?

My [image](https://github.com/retropex/bitcoin/pkgs/container/bitcoin) is multi-arch so I don't understand why this is not working.

**Comment by luke-jr** [comment]
Is this still an issue for 28.1?

**Comment by Retropex** [comment]
I think the Umbrel team would have noticed if there was a problem with 28.1 since they test before releasing, so if we have no news from @mokeybear I think you can close the issue.
